src/SFA.DAS.EmployerAccounts/Queries/GetAccountLegalEntitiesByHashedAccountId/GetAccountLegalEntitiesByHashedAccountIdResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using SFA.DAS.EmployerAccounts.Models.Account;
+
+namespace SFA.DAS.EmployerAccounts.Queries.GetAccountLegalEntitiesByHashedAccountId
+{
+    public class GetAccountLegalEntitiesByHashedAccountIdResponse
+    {
+        public List<AccountSpecificLegalEntity> Entities { get; set; }
```

Comment:
Can this be renamed to LegalEntities?

Reply:
It can and just has.

I had already changed it from `Entites` :0)

I've updated the same property name on the original `GetLegalEntitesResponse` as well.